scripts/tests/doppelganger_protection.sh

```diff
@@ -144,7 +142,10 @@ if [[ "$BEHAVIOR" == "success" ]]; then
 
     # Cleanup
     cd $PREVIOUS_DIR
-    kill $BOOT_PID $BEACON_PID $BEACON_PID2 $BEACON_PID3 $GANACHE_PID $VALIDATOR_1_PID $VALIDATOR_2_PID $VALIDATOR_3_PID $VALIDATOR_4_PID
+
+    killall geth
+    killall lighthouse
+    killall bootnode
```

Comment:
I think the PID is a bit of pain and happy to see it go, but with `killall` we have to assume users don't run these on their staking machines, probably unlikely... but just a thought 🤔

Reply:
I would really hope that no one is running doppelganger tests on their staking machines. The alternative of storing PIDs and killing them one by one is a pain imo. I'm happy to remove the `killall` if that is better since it doesn't really matter in CI and we could manually kill the processes when testing out changes locally.

Reply:
Another alternative would be killing the processes by port - in that case we'll probably need to expose a port for VC (`http` is exposed for VC in Siren for testing), but I'm not too fuss as this is only used in doppelganger tests

```
sudo lsof -t -i :$PORT | xargs kill -9
```